### DISCUSSION TITLE
<title> i'm studying yolov5 and I have a question.

### DISCUSSION THREAD

**Discussion by whereismylife** [community/community#44635] [Discussions] [answered]
### Select Topic Area

Question

### Body

Hi!

I'm studying about yolov5 and I am having fun using yolov5. and, I'm labeling using a program called labelme. As I was using labelme, I suddenly had this thought. I'd like to know which one is more accurate, between labeling with a square or drawing an outline.
I know that it's done by pixel, but I'm not sure exactly. I would appreciate it if you could tell me whether it is better to use a square or an outline.

**Comment by MihirRajeshPanchal** [comment, marked as answer]
The choice between using a square or an outline to label objects in an image depends on the specific use case and the requirements of your object detection model.

Using a square to label an object is generally considered to be more accurate, as it provides a clear and precise boundary for the object. This can be useful in situations where the object is well-defined and clearly visible, and where the model needs to be able to accurately identify the object in various positions and orientations.

On the other hand, using an outline to label an object can be more flexible, as it allows for a more nuanced representation of the object's shape. This can be useful in situations where the object is less well-defined or partially obscured, and where the model needs to be able to detect the object despite variations in its appearance.

Ultimately, it's best to experiment with both square and outline labeling and compare the performance of the resulting model to see which one is more accurate in your specific case.

**Reply by whereismylife** [reply]
thank you so much about your answer. I hope you are full of happiness for helping me.
